compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp

```diff
@@ -17,10 +17,203 @@
+    auto input = g->nodes()->create<luci::CircleConst>();
+    input->dtype(loco::DataType::FLOAT32);
+    input->shape({1, 3});
+    input->size<loco::DataType::FLOAT32>(3);
+    input->at<loco::DataType::FLOAT32>(0) = 1;
+    input->at<loco::DataType::FLOAT32>(1) = 2;
+    input->at<loco::DataType::FLOAT32>(2) = 3;
+
+    // CircleConst (BroadcastTo-shape)
+    auto shape = g->nodes()->create<luci::CircleConst>();
+    shape->dtype(loco::DataType::S32);
+    shape->shape({2});
+    shape->size<loco::DataType::S32>(2);
+    shape->at<loco::DataType::S32>(0) = 2;
+    shape->at<loco::DataType::S32>(1) = 3;
+
+    // Custom BroadcastTo
+    _custom_broadcastTo = g->nodes()->create<luci::CircleCustom>(2, 1);
+    _custom_broadcastTo->custom_code("BroadcastTo");
+    _custom_broadcastTo->dtype(loco::DataType::FLOAT32);
+    _custom_broadcastTo->shape({2, 3});
+    _custom_broadcastTo->name("BroadcastTo");
+
+    _custom_broadcastTo->inputs(0, input);
+    _custom_broadcastTo->inputs(1, shape);
+
+    _custom_broadcastTo_out = g->nodes()->create<luci::CircleCustomOut>();
+    _custom_broadcastTo->custom_code("BroadcastTo");
+    _custom_broadcastTo_out->shape({2, 3});
+    _custom_broadcastTo_out->dtype(loco::DataType::FLOAT32);
+    _custom_broadcastTo_out->index(0);
+    _custom_broadcastTo_out->input(_custom_broadcastTo);
+
+    // Builtin BroadcastTo
+    _builtin_broadcastTo = g->nodes()->create<luci::CircleBroadcastTo>();
+    _builtin_broadcastTo->dtype(loco::DataType::FLOAT32);
+    _builtin_broadcastTo->name("BroadcastTo");
+    _builtin_broadcastTo->input(input);
+    _builtin_broadcastTo->shape(shape);
+
+    // CircleCustomOut
+    _addV2_out = g->nodes()->create<luci::CircleCustomOut>();
+    _addV2_out->shape({2, 3});
+    _addV2_out->dtype(loco::DataType::FLOAT32);
+    _addV2_out->index(0);
+    _addV2_out->input(_addV2);
+  }
+
+public:
+  luci::CircleCustom *addV2() { return _addV2; }
+  luci::CircleCustom *custom_broadcastTo() { return _custom_broadcastTo; }
+  luci::CircleCustomOut *custom_broadcastTo_out() { return _custom_broadcastTo_out; }
+  luci::CircleBroadcastTo *builtin_broadcastTo() { return _builtin_broadcastTo; }
+
+protected:
+  luci::CircleCustom *_addV2 = nullptr;
+  luci::CircleCustomOut *_addV2_out = nullptr;
+  luci::CircleCustom *_custom_broadcastTo = nullptr;
+  luci::CircleBroadcastTo *_builtin_broadcastTo = nullptr;
+  luci::CircleCustomOut *_custom_broadcastTo_out = nullptr;
+};
+
+class BroadcastToAddV2Graph : public TestIGraphlet,
+                              public TestOsGraphlet<1>,
+                              public BroadcastToAddGraphlet
+{
+public:
+  BroadcastToAddV2Graph() = default;
+
+  void init(void)
+  {
+    TestIGraphlet::init(g(), {2, 3});
+    TestOsGraphlet<1>::init(g(), {{2, 3}});
+    BroadcastToAddGraphlet::init(g());
+
+    // connect graph
+    _addV2->inputs(0, input());
+
+    output(0)->from(_addV2_out);
+  }
+};
+
+class PassTest : public ::testing::Test
```

Comment:
plz give a name for this test not `PassTest`

Reply:
@seanshpark 

okay.. I set `ResolveCustomOpAddPassTest` as the name in my local,
but the name was already used in the unit test in [here](https://github.com/Samsung/ONE/blob/31295c97fc8f10e630c105a31f131683a9ad4548/compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp#L21).
so, a conflict occurred in gtest. 😅
How about temporarily setting the name to `PassTest` in this PR,
and modifying it by uploading another PR in following order..?

1) Modify unit name test from `TEST` to `TEST_F` using `PassTest`. -> [here](https://github.com/Samsung/ONE/blob/31295c97fc8f10e630c105a31f131683a9ad4548/compiler/luci/pass/src/ResolveCustomOpAddPass.test.cpp#L21)

2) Change the name of `PassTest` to `ResolveCustomOpAddPassTest` and the related parts.

Reply:
> How about temporarily setting the name to PassTest in this PR,

Fire a PR that changes existing one to something else if you want to keep `ResolveCustomOpAddPassTest` for the new ones.

Reply:
@seanshpark 

Okay !! I think the method you provided is less complicated, 
so I'll change the existing one first.
Thank you a lot !! :)